alivee/place_search_engine/utils.py

```diff
@@ -0,0 +1,139 @@
+# Put Standard Library Imports Here:
+import math
+import time
+
+# Put Third Party/Django Imports Here:
+from geopy.distance import vincenty
+
+# Put Alivee Imports Here:
+from general_utils.requests.google.place_search.constants import PlaceSearchPlaceType
+from general_utils.requests.google.place_search.exceptions import ExceedMaximumMainRequestRetryCountException
+from general_utils.requests.google.place_search.utils import PlaceSearch
+from place_search_engine.constants import CityBoundaryPointsInfo
+
+MAX_BACKFILL_RADIUS_IN_METERS = 1000
```

Comment:
Still a too large value for some of the restaurant-crowded area such as DOWNTOWN